slides/installation.qmd

```diff
@@ -4,8 +4,8 @@
 #### Silicon mac users
 Install brainglobe tools individually:
```

Comment:
```suggestion
```

Reply:
The equivilant sentence can probably be removed in the other section too.

Reply:
Done!